app/core/global-styles/inputs.scss

```diff
@@ -0,0 +1,29 @@
+@import "colors";
+
+.form-input {
+  width: 98%;
+  margin: 0.5rem 0;
+  padding: 0.25rem;
+  height: 1.5rem;
+  border: 0.2rem solid $grayLightColor;
+  border-radius: 0.25rem;
+}
+
+.from-textarea {
+  width: 98%;
+  margin: 0.5rem 0;
+  padding: 0.25rem;
+  border: 0.2rem solid $grayLightColor;
+  border-radius: 0.25rem;
+  font-family: 'Roboto', sans-serif;
+}
+
+.login-form-input {
+  width: 98%;
+  margin-top: -0.5rem;
+  margin-bottom: 0.5rem;
+  padding: 0.25rem;
+  height: 1.5rem;
+  border: 0.2rem solid $grayLightColor;
+  border-radius: 0.25rem;
```

Comment:
Please take a look at this file and move duplicated rules to mixins.

Reply:
added new mixin